### DISCUSSION TITLE
I want to build a model with neper that I can import into abaqus

### DISCUSSION THREAD

**Discussion by tage-123** [neperfepx/neper#881] [Q&A] [answered]
I want to build a model of 100 cells where the orientation of each cell is alternating between alpha orientation and λ orientation and make it a mapped grid model and export it to an inp file how do I do that

**Comment by rquey** [comment, marked as answer]
To build a simple model:
```
neper -T -n 100
```
To mesh it into a mapped grid for Abaqus:
```
neper -M n100-id1.tess -elt hex -for inp
```
To load orientations from a file, use option [`-ori file(...)`](https://neper.info/doc/neper_t.html#cmdoption-ori) (in the -T command).
